infoblox-ipam.go

```diff
@@ -83,17 +94,12 @@ func (ibDrv *InfobloxDriver) RequestAddress(netviewName string, cidr string, ipA
 	return fmt.Sprintf("%s", fixedAddr.IPAddress), nil
 }
 
-func (ibDrv *InfobloxDriver) UpdateAddress(netviewName string, cidr string, ipAddr string, macAddr string, vmID string) (string, error) {
-        var fixedAddr *ibclient.FixedAddress
-
-	if netviewName == "" {
-		netviewName = ibDrv.DefaultNetworkView
-	}
+func (ibDrv *InfobloxDriver) UpdateAddress(fixedAddr *ibclient.FixedAddress, macAddr string, vmID string) (string, error) {
 
-        fixedAddr, _ = ibDrv.objMgr.UpdateFixedAddress(netviewName, cidr, ipAddr, macAddr, vmID)
+	fixedAddr, _ = ibDrv.objMgr.UpdateFixedAddress(fixedAddr, macAddr, vmID)
 
-        log.Printf("UpdateAddress: fixedAddr result is '%s'", *fixedAddr)
-        return fmt.Sprintf("%s", fixedAddr.IPAddress), nil
+	log.Printf("UpdateAddress: fixedAddr result is '%s'", *fixedAddr)
+	return fmt.Sprintf("%s", fixedAddr.IPAddress), nil
```

Comment:
It doesn't seem very useful for a "update" method to return a value that is not one that is being updated. Maybe it should return the `FixedAddress` object reflects the result of the update?